src/cs/MonoGame.Extended.Collisions/CollisionComponent.cs

```diff
@@ -97,6 +98,50 @@ public override void Update(GameTime gameTime)
             }
         }
 
+        /// <summary>
+        /// Cast an actor's colliders in a direction and check for collisions.
+        /// </summary>
+        /// <param name="actors">The list of actor colliders to cast.</param>
+        /// <param name="move">Normalized vector representing the direction to cast each shape.</param>
+        /// <param name="hitBuffer">Buffer to receive results.</param>
+        /// <param name="distance">Maximum distance over which to cast the Collider(s).</param>
+        public int Cast(IEnumerable<ICollisionActor> actors, Vector2 move, List<CollisionEventArgs> hitBuffer, float distance)
+        {
+            hitBuffer.Clear();
```

Comment:
Does it matter that hitBuffer.Clear() isn't called in the 2nd Cast?  What if someone calls the 2nd public method instead, then the hitBuffer won't get cleared right?